### ISSUE TITLE
P100 logged-in but not selectable/saveable inside item - why?

### ISSUE THREAD

**Issue by redskate** [wildcs/oh3_tapocontrol#5]
Dear community

I am running an openHAB 3.3.0 Docker on OSX Mojave with 3.3.0 tapo bindings (downloaded inside OH3).
OH3 items cannot link with (successfully cloud-logged-in) P100 things (no save link offered after P100 selection).

What could I am doing wrong here? 

How can I manage to get the P100 thing saved inside an item (in order to switch it on and off)?

Thanks a lot for your kind response

**Comment by wildcs** [comment]
Did you set the correct IP in the thing settings?
Whats the state of the item and what is about the thing in the logfile?

**Comment by redskate** [comment]
Thank you for that. Yes I set the internal subnet IP of that Tapo Control in the panel. That control has a static DHCP and once connected, it gets its IP in the subnet.

**Comment by wildcs** [comment]
Can you ping the device with the IP you Set in the Thing-Settings?
Pleased Post Screenshot of Thing-Settings and the Log Message belonging to TapoControl.

**Comment by redskate** [comment]
I can issue a netcat command from INSIDE the docker container:

nc -v x.x.x.101 80

for port 80 (I guess) and the answer is: "Connection to x.x.x.101 80 port [tcp/http] succeeded!

**Comment by wildcs** [comment]
Thing ist online so communication seems to be okay.

**Comment by redskate** [comment]
Yes, but in the item, after selection ot that thing, non "save" button is offered - so item and thing are not bound ...

**Comment by redskate** [comment]
In the item panel, after selection, on the upper right, no "save" ...  and then I can switch the item on and off as often I want ... nothing happens on the tape control.


![Screenshot 2022-09-07 at 16 12 04](https://user-images.githubusercontent.com/977837/188900281-ba2c66ac-df1c-4ed2-9cb6-72bc5deee530.png)

**Comment by wildcs** [comment]
How do you try to link the item to thing? Looks wired and not as a binding bug.

In my OH3.3.0 it looks like this:
![image](https://user-images.githubusercontent.com/40909464/189032433-7fc58a14-fc2c-4d45-bfbd-555aad4af9f3.png)
![image](https://user-images.githubusercontent.com/40909464/189032598-c1bb3418-f8e9-43ad-9cc0-071516bfc4b9.png)
![image](https://user-images.githubusercontent.com/40909464/189032674-089edb2c-5d87-4593-ba1a-d62b4c24885f.png)

And then the Item looks like this:
![image](https://user-images.githubusercontent.com/40909464/189032755-4285ef0b-5c65-4186-ba7d-6fba59ecbe5d.png)

**Comment by redskate** [comment]
I discovered one missing action and I did that: adding an item one has to determine the class of the item, which is default a measurement; here I changed the class to "SWITCH" and pressed "save". Again I get a measurement Item ... so I must du again something wrong.
![Screenshot 2022-09-08 at 13 14 19](https://user-images.githubusercontent.com/977837/189108369-e64c5a06-2e50-4236-8d11-11b7fb7ed611.png)

**Comment by redskate** [comment]
I managed to link a switch inside the thing panel under the specific switch place. And ... as soon as you do the right thing, it functions! Thank you very much!!!

At this point I haveto say, I tried to follow the manual developing a model top down, i.e. from model to thing and I did not success. Maybe there is there some need of improvement ??? 

THANK YOU :)
![Screenshot 2022-09-08 at 13 21 37](https://user-images.githubusercontent.com/977837/189109881-8c01c272-153d-43ce-99bb-e1718df89a70.png)
